operator.yaml

```diff
@@ -26,3 +26,7 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
+            - name: POD_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
```

Comment:
Why does the Operator need to know his name?

Reply:
operators now assume this https://github.com/operator-framework/operator-sdk/blob/45c93917cf876ed15c0601f0036d91f29324919f/pkg/k8sutil/k8sutil.go#L65